README.md

```diff
@@ -170,15 +123,6 @@ following keys:
 | `sequences_per_page` | Optional - Number of [Mapillary sequences](https://www.mapillary.com/developer/api-documentation/#sequences) that should be pulled in each API call |
 | `skip_if_fewer_images_than` | Optional - Skip a Mapillary sequence if it has fewer Mapillary images than this value |
 
-Here is an example setup and run across a wide area in Manhattan NYC, with Mapillary client ID redacted (assuming you
-have Python 3.9 installed using `python3`):
-
-```bash
-python3 -m venv venv
-source venv/bin/activate
-pip install -r requirements.txt
```

Comment:
haha yeah i didnt see it at the end of the section, i assumed it would be at the top, my mistake!

Reply:
Yep, also realized it should probably be at the top so I moved it there :)